tests/pruning/helpers.py

```diff
@@ -39,16 +39,16 @@ class TestModelDiffConvs(nn.Module):
     def __init__(self):
         super().__init__()
         # Usual conv
-        self.conv1 = create_conv(1, 3, 2, 9, -2)
+        self.conv1 = create_conv(1, 32, 2, 9, -2)
```

Comment:
Why do you need to change shapes here?

Reply:
To check shapes in pruned module, number of remaining channels should be multiple of 8.